flux_local/kustomize.py

```diff
@@ -251,7 +251,7 @@ async def fluxtomize(path: Path) -> bytes:
     filenames.sort()
     for filename in filenames:
         new_path = path / filename
-        if new_path.is_dir() and await can_kustomize_dir(new_path):
+        if new_path.is_dir():
```

Comment:
This is the only purpose of `can_kustomize_dir` so can you remove it and the test?

Reply:
This is still used in `Build.run` and as far as I understand this is still needed to decide if the kustomize or fluxtomize should be used

https://github.com/allenporter/flux-local/blob/main/flux_local/kustomize.py#L209
```python
        if not await can_kustomize_dir(self._path):
            # Attempt to effectively generate a kustomization.yaml on the fly
            # mirroring the behavior of flux
            return await fluxtomize(self._path)
```

Reply:
Ah! thank you I missed that.